server/controllers/cv-controller.js

```diff
@@ -0,0 +1,44 @@
+const client = require('../lib/github-client');
+
+console.log(client.getUserProfile());
+function fetchingData(username, res) {
+	const userProfile = client.getUserProfile(username);
+	const PullRequest = client.getUserPullRequests(username);
+	return Promise.all([userProfile, PullRequest])
+  .then((data
+  	) =>{
+  	console.log(data);
+    return res.render('index', {code:data[0], codes:data[1][0]});
```

Comment:
Could you give the properties you send to the template more descriptive names than `code` and `codes`?